### PR TITLE
Add DCR URL to Ipsos test article

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

Adds DCR URL to the Ipsos test.

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const init = (): Promise<void> => {
         onConsentChange(state => {
             // Initial testing only
             console.log(getConsentFor('ipsos', state));
-            if(document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3") {
+            if(document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3" || document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3?dcr=true") {
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js:28]: I think in this instance you may want to simply target the `pathname`.

```suggestion
            if(document.location.pathname === "/science/grrlscientist/2012/aug/07/3") {
```

**Review by mxdvl — APPROVED** [review]
Looks fine, but you may simply want to ignore the query strings.

File: static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js (hunk 1 above)
Comment (nitro-marky) [reply static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js]: Updated  👍

File: static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js [hunk 2]
```diff
@@ -25,10 +25,12 @@ export const init = (): Promise<void> => {
         onConsentChange(state => {
             // Initial testing only
             console.log(getConsentFor('ipsos', state));
-            if(document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3") {
-                if (getConsentFor('ipsos', state)) {
-                    loadIpsosScript();
-                }
+            if(document.location.pathname === "/science/grrlscientist/2012/aug/07/3" || document.location.pathname === "/science/grrlscientist/2012/aug/07/3?dcr=true")
```
Comment (jfsoul) [inline static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js:28]: `location.pathname` just includes the path, no query string, so you can remove the second URL check here I believe.

Comment (nitro-marky) [reply static/src/javascripts/projects/commercial/modules/ipsosTaggingScript.js]: Yep, you're right. Updated 👍

**Review by jfsoul — APPROVED** [review]

**Pull request merged**